lib/Fmt_ast.ml

```diff
@@ -4320,7 +4320,7 @@ and fmt_value_binding c ~rec_flag ?ext ?in_ ?epi
   in
   let cstr_indent =
     match lb_typ with
-    | `Other {ast= {ptyp_desc= Ptyp_poly _; _}; _} -> 6
+    | `Other {ast= {ptyp_desc= Ptyp_poly _; _}; _} -> 3
```

Comment:
I think this should be 4. The indentation of 1 compared to the function body feels weird. Also, that's the indentation of other kinds of type annotations, which would remove the special case entirely.

Reply:
I changed the indent to 4 in the last commit to have a look, looks good to me as well, but 6 was a bit excessive.

Reply:
The match and the variable can be removed entirely now.